WORKSPACE

```diff
@@ -139,7 +139,28 @@ go_register_toolchains(
     version = "1.16",
 )  # airy_nogo is in the top-level BUILD file of this workspace
 
-load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
+load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies", "go_repository")
```

Comment:
Sometimes gazelle does not seem to pick up the `to_macro` directive, which tells it to write these to the `go_repositories.bzl`. I fixed this for the `update-deps` command on `develop`. So if you remove this, rebase and run `update-deps` they should show up in the repositories file :)